src/mongo/shell/utils.js

```diff
@@ -849,6 +849,12 @@ shellHelper.it = function() {
     shellPrintHelper(___it___);
 };
 
+shellHelper.mongo = function() {
+    print("You've typed the mongo command within a mongo shell.");
+    print("This is an error.");
+    print("If you meant to run a new mongo shell, please exit the current shell.");
```

Comment:
@terakilobyte thought: We could add a dochub link here to point users to https://docs.mongodb.com/manual/crud/ ,so they immediately have a point of reference to start using the shell.